lib/generators/base.rb

```diff
@@ -74,7 +74,13 @@ def env_as_assigns(env)
     def run(command, env={})
       command = "rvm #{ruby_version} do #{command} >/dev/null"
       log "#{env_as_assigns(env)} #{command}"
-      system(env, command)
+
+      if system(env, command)
+        true
+      else
+        log "\"#{command}\" failed to execute"
+        exit
+      end
```

Comment:
This could probably be included in the `Running` module with `env_as_assigns` there too.